lib/from_honeybee/_openapi/honeybee_workflow.osw

```diff
@@ -0,0 +1,47 @@
+{
+  "seed_file": null,
+  "weather_file": null,
+  "measure_paths": [],
+  "file_paths": [],
+  "run_directory": null,
+  "steps": [
+    {
+      "measure_dir_name": "from_honeybee_simulation_parameter",
+      "arguments": {
+        "simulation_parameter_json": "multi_zone_model.json"
```

Comment:
I know this osw is just as an example since we will write our own out from the Python end but can we change this line to something more meaningful like:
```
"simulation_parameter_json": "simulation_parameter.json"
```